### PR TITLE
Better readability(Personal opinion).

### PR DESCRIPTION
**Pull request by zhangenming**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/functions/default_parameters/index.md [hunk 1]
```diff
@@ -96,8 +96,8 @@ test(null)       // 'object' (num is set to null)
 
 ### Evaluated at call time
 
-The default argument is evaluated at _call time_. So, unlike (for example)
-Python, a new object is created each time the function is called.
+The default argument is evaluated at _call time_. unlike (for example) Python,
+So, a new object is created each time the function is called at here.
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/functions/default_parameters/index.md:100]: I think this is even better.

```suggestion
So, a new object is created each time the function is called.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/javascript/reference/functions/default_parameters/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/javascript/reference/functions/default_parameters/index.md:100]: Yes, this is better. But the whole thing needs work. Hold on.

Comment (hamishwillee) [inline files/en-us/web/javascript/reference/functions/default_parameters/index.md:100]: Here we go.
```suggestion
The default argument is evaluated at _call time_.
Unlike with Python (for example), a new object is created each time the function is called.
```

**Review by hamishwillee — APPROVED** [review]
Thanks @zhangenming - good catch - the existing text definitely needed work. @teoli2003 I modified this, so you might want to re-review.

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Pull request merged**